supply_energy.md

```diff
@@ -0,0 +1,291 @@
+
+See `annualsupply` in [smooth_renewable_subresource.cpp](https://github.com/JGCRI/gcam-core/blob/master/cvs/objects/resources/source/smooth_renewable_subresource.cpp).
+
+## Policy options 
+This section summarizes some of the energy-based policy options available in GCAM. 
+
+### Energy production policies
+
+Users can include [energy production policies](policies.html#energy-production-policies), which can constrain energy production or create incentives to increase or decrease production. 
+
+### Energy intensity standards
+
+Users can also include [energy intensity standards](policies_examples.html#res), such as a renewable energy standard.
+
+## Insights and intuition
+
+### Paper/Topic #1
+
+{One paragraph summary of a key insight from one or more papers}
+
+## IAMC Reference Card
+
+### Energy technology substitution
+
+Energy technology choice
+- [ ] No discrete technology choices
+- [X] Logit choice model
+- [ ] Production function
+- [ ] Linear choice (lowest cost)
+- [ ] Lowest cost with adjustment penalties
+
+Energy technology substitutability
+- [ ] Mostly high substitutability
+- [ ] Mostly low substitutability
+- [X] Mixed high and low substitutability
+
+Energy technology deployment
+- [ ] Expansion and decline constraints
+- [X] System integration constraints
+
+### Energy
+
+Electricity technologies
+- [X] Coal w/o CCS
+- [X] Coal w/ CCS
+- [X] Gas w/o CCS
+- [X] Gas w/ CCS
+- [X] Oil w/o CCS
+- [X] Oil w/ CCS
+- [X] Bioenergy w/o CCS
+- [X] Bioenergy w/ CCS
+- [X] Geothermal power
+- [X] Nuclear power
+- [X] Solar power
+- [X] Solar power-central PV
+- [X] Solar power-distributed PV
+- [X] Solar power-CSP
+- [X] Wind power
+- [X] Wind power-onshore
+- [X] Wind power-offshore
+- [X] Hydroelectric power
+- [ ] Ocean power
+
+Hydrogen production
+- [X] Coal to hydrogen w/o CCS
+- [X] Coal to hydrogen w/ CCS
+- [X] Natural gas to hydrogen w/o CCS
+- [X] Natural gas to hydrogen w/ CCS
+- [X] Oil to hydrogen w/o CCS
+- [X] Oil to hydrogen w/ CCS
+- [X] Biomass to hydrogen w/o CCS
+- [X] Biomass to hydrogen w/ CCS
+- [X] Nuclear thermochemical hydrogen
+- [X] Solar thermochemical hydrogen
+- [X] Electrolysis
+
+Refined liquids
+- [X] Coal to liquids w/o CCS
+- [X] Coal to liquids w/ CCS
+- [X] Gas to liquids w/o CCS
+- [X] Gas to liquids w/ CCS
+- [X] Bioliquids w/o CCS
+- [X] Bioliquids w/ CCS
+- [X] Oil refining
+
+Refined gases
+- [X] Coal to gas w/o CCS
+- [X] Coal to gas w/ CCS
+- [X] Oil to gas w/o CCS
+- [X] Oil to gas w/ CCS
+- [X] Biomass to gas w/o CCS
+- [X] Biomass to gas w/ CCS
+
+Heat generation
```

Comment:
This one is a bit unclear to me.  Are we referring to district heat here, or all heat (which I guess would include district heat + cogen).  In either case, I think Geothermal heat and Solarthermal heat should be removed.  If we're including industrial cogen here, we have a hydrogen cogen technology option as well.